content/en/docs/contribute/participate/roles-and-responsibilties.md

```diff
@@ -188,4 +188,10 @@ To apply:
 
 2. Assign the PR to one or more current SIG Docs approvers.
 
-If approved, a SIG Docs lead adds you to the appropriate GitHub team. Once added, [K8s-ci-robot](https://github.com/kubernetes/test-infra/tree/master/prow#bots-home) assigns and suggests you as a reviewer on new pull requests.
\ No newline at end of file
+If approved, a SIG Docs lead adds you to the appropriate GitHub team. Once added, [K8s-ci-robot](https://github.com/kubernetes/test-infra/tree/master/prow#bots-home) assigns and suggests you as a reviewer on new pull requests.
+
+## {{% heading "whatsnext" %}}
+
+Learn about specialized roles in SIG Docs:
```

Comment:
Possibly:
```suggestion
Learn about specialized roles in SIG-Docs:
```

Reply:
Nope. We are sticking to 'SIG Docs'.
https://kubernetes.io/docs/contribute/style/style-guide/#kubernetes-io-word-list